config/initializers/geocoder.rb

```diff
@@ -1,14 +1,14 @@
 Geocoder.configure(
   # geocoding options
   # :timeout      => 3,           # geocoding service timeout (secs)
-   #:lookup       => :bing,     # name of geocoding service (symbol)
-  :lookup       => :google,  
+  :lookup       => :bing,     # name of geocoding service (symbol)
+  # :lookup       => :google,  
   # :language     => :en,         # ISO-639 language code
    #:use_https    => true,       # use HTTPS for lookup requests? (if supported)
   # :http_proxy   => nil,         # HTTP proxy server (user:pass@host:port)
   # :https_proxy  => nil,         # HTTPS proxy server (user:pass@host:port)
-   #:api_key      => ENV['bing_map_key']      # API key for geocoding service
-  :api_key => ENV['google_map_key']
+   :api_key      => ENV['bing_map_key']      # API key for geocoding service
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.